checks/raw/license.go

```diff
@@ -387,25 +387,8 @@ type fsfOsiLicenseType struct {
 	Name string
 }
 
-// (there's no magic here)
-// created from the table at https://spdx.org/licenses
-// 1) convert table to CSV and convert to json
-// (inspired from inspired from https://stackoverflow.com/questions/17912307/u-ufeff-in-python-string)
-//
-//	cat SPDX-license-IDs-202211131030CST.csv | \
-//	 python -c \
-//	   'import csv, json, sys; print(json.dumps([dict(r) for r in csv.DictReader(sys.stdin)]))' | \
-//	 sed 's/\\ufeff//g' > SPDX-license-IDs-202211131030CST.json
-//
-// 2) convert json to go lang compatible map
-//
-//	cat SPDX-license-IDs-202211131030CST.json | \
-//	 jq -c '.[] | select((."FSF Free/Libre?"=="Y") or (."OSI Approved?"=="Y")) | \
-//	   [."Identifier",."Full Name"]' | \
-//	 sed 's/","/": \{ Name: "/;s/\["/"/;s/"\]/" },/' | \
-//	 sed 's/[- ]only//g;s/[- ]or[- ]later//g' | \
-//	 sort | \
-//	 uniq
+// SPDX license list available here: https://spdx.org/licenses (OSI approved / FSF licenses only)
+// JSON formatted list: https://github.com/spdx/license-list-data/blob/main/json/licenses.json
 var fsfOsiApprovedLicenseMap = map[string]fsfOsiLicenseType{
```

Comment:
I started working on a CLI tool that generates the list of licenses in a Go map-friendly format, but was planning to file that as a separate PR so it doesn't hold up the license list update. I'd be happy to re-add the `curl` comment in the interim, with a note that sorting may not work as expected since the datasource has changed?

Reply:
I think an interim comment would be great just in case the tool falls off.

Reply:
Added back!